RecoLuminosity/LumiProducer/test/BuildFile.xml

```diff
@@ -1,5 +1,26 @@
-<use name="FWCore/Framework"/>
-<use name="DataFormats/Luminosity"/>
+<use   name="FWCore/Framework"/>
+<use   name="DataFormats/Luminosity"/>
+
+<library file="analysis/plugins/*.cc" name="PCCNTupler">
+    <use   name="FWCore/PluginManager"/>
+    <use   name="FWCore/ParameterSet"/>
+    <use   name="DataFormats/VertexReco"/>
+    <use   name="DataFormats/DetId"/>
+    <use   name="Geometry/TrackerGeometryBuilder"/>
+    <use   name="DataFormats/SiPixelDetId"/>
+    <use   name="DataFormats/TrackerRecHit2D"/>
+    <use   name="DataFormats/TrackingRecHit"/>
+    <use   name="HLTrigger/HLTcore"/>
+    <use   name="Alignment/OfflineValidation"/>
+    <use   name="CondFormats/SiPixelObjects"/>
+    <use   name="CondFormats/RunInfo"/>
+    <use   name="clhep"/>
```

Comment:
Alignment/OfflineValidation and CondFormats/SiPixelObjects are not needed according to the libchecker